demo/example.html

```diff
@@ -0,0 +1,52 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge,chrome=1">
+  <title>Editor</title>
+  <style type="text/css" media="screen">
+    body {
+        overflow: hidden;
+    }
+    #editor {
+        border: 1px solid lightgray;
+        height: 600px;
+        width: 99%;
+    }
+   #maket {
+    width: 100%; 
+   }
+   #maket TD {
+    vertical-align: top; 
+   }
+   TD#leftcol {
+    width: 80%; 
+   }
+  </style>
+</head>
+<body>
+<table cellspacing="0" cellpadding="0" id="maket">
+   <tr> 
+    <td id="leftcol">
+      <pre id="editor">
+    \documentclass[12pt]{article}
+
+    \usepackage{answers}
+    \usepackage{setspace}
+    \usepackage{graphicx}
+}</pre>
+
+<script src="src-noconflict/ace.js" charset="utf-8"></script>
+<script>
```

Comment:
Normally you don't want to mix `script` tags with other HTML markup. Move it to the end of document, right before the closing `</html>`